tests/phpunit/integration/api/actions/beansModifyAction.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Tests for beans_modify_action()
+ *
+ * @package Beans\Framework\Tests\Integration\API\Actions
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Integration\API\Actions;
+
+use Beans\Framework\Tests\Integration\API\Actions\Includes\Actions_Test_Case;
+
+require_once __DIR__ . '/includes/class-actions-test-case.php';
+
+/**
+ * Class Tests_BeansModifyAction
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
```

Comment:
I noticed that multiple package names are incorrect in the integration tests files, probably due to copy/paste from the Unit tests. Could you kindly review all package names please?